demisto_sdk/commands/content_graph/common.py

```diff
@@ -0,0 +1,169 @@
+
+import enum
+import os
+from pathlib import Path
+from typing import Any, Dict, Iterator, List, Set
+from demisto_sdk.commands.common.tools import get_content_path
+
+REPO_PATH = Path(get_content_path())
+
+NEO4J_ADMIN_DOCKER = ''
+
+NEO4J_DATABASE_URL = os.getenv('DEMISTO_SDK_NEO4J_DATABASE_URL', 'bolt://localhost:7687')
+NEO4J_USERNAME = os.getenv('DEMISTO_SDK_NEO4J_USERNAME', 'neo4j')
+NEO4J_PASSWORD = os.getenv('DEMISTO_SDK_NEO4J_USERNAME', 'test')
+
+
+PACKS_FOLDER = 'Packs'
+PACK_METADATA_FILENAME = 'pack_metadata.json'
+UNIFIED_FILES_SUFFIXES = ['.yml', '.json']
+
+
+class Rel(enum.Enum):
+    USES = 'USES'
+    USES_COMMAND_OR_SCRIPT = 'USES_COMMAND_OR_SCRIPT'
+    DEPENDS_ON = 'DEPENDS_ON'
+    TESTED_BY = 'TESTED_BY'
+    IN_PACK = 'IN_PACK'
+    HAS_COMMAND = 'HAS_COMMAND'
+    IMPORTS = 'IMPORTS'
+
+    def __str__(self):
+        return self.value
+
+
+class ContentTypes(str, enum.Enum):
+    BASE_CONTENT = 'BaseContent'
+    PACK = 'Pack'
+    COMMAND_OR_SCRIPT = 'CommandOrScript'
+    COMMAND = 'Command'
+    SCRIPT = 'Script'
+    PLAYBOOK = 'Playbook'
+    INTEGRATION = 'Integration'
+    TEST_PLAYBOOK = 'TestPlaybook'
+    INCIDENT_FIELD = 'IncidentField'
+    INCIDENT_TYPE = 'IncidentType'
+    INDICATOR_TYPE = 'IndicatorType'
+    INDICATOR_FIELD = 'IndicatorField'
+    CLASSIFIER = 'Classifier'
+    MAPPER = 'Mapper'
+    LAYOUT = 'Layout'
+    WIDGET = 'Widget'
+    DASHBOARD = 'Dashboard'
+    REPORT = 'Report'
+    CONNECTION = 'Connection'
+    GENERIC_DEFINITION = 'GenericDefinition'
+    GENERIC_FIELD = 'GenericField'
+    GENERIC_MODULE = 'GenericModule'
+    GENERIC_TYPE = 'GenericType'
+    LIST = 'List'
+    PREPROCESS_RULE = 'PreProcessRule'
+    JOB = 'Job'
```

Comment:
Is this list ordered in some way?
`JOB` is after `PREPROCESS_RULE`.

Reply:
Job is a scheduler for a playbook, preprocess rule is a way to modify incident data before it's created (or prevent its creation). AFAIK preprocess rules are not really supported by the server.